examples/cli/main.rs

```diff
@@ -1,3 +1,6 @@
+#![forbid(unsafe_code)]
```

Comment:
Does this work at file level or do we need it only in the entrypoints?

Reply:
I think it applies to the whole crate, so yeah just one for main, one for lib, and one for each example.